OpenLaTeXStudio/EditorModule/nbproject/project.xml

```diff
@@ -105,6 +105,14 @@
                     </run-dependency>
                 </dependency>
                 <dependency>
+                    <code-name-base>org.openide.util.ui</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>9.4.1</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>

```

Comment:
It contains the HelpCtx class (used in the LaTeX settings controller); my NetBeans was giving a warning for this with suggested resolution to add this dependency. I took the liberty to add it.